src/Maker/MakeAuthenticator.php

```diff
@@ -209,6 +210,15 @@ function ($answer) {
                     $supportRememberMeValues[$supportRememberMeType]
                 );
             }
+
+            $command->addArgument('support-throttling', InputArgument::OPTIONAL);
```

Comment:
`MakeAuthenticator` is being deprecated - we are not adding any new features to it.